.github/workflows/release.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Fix the release notes, it was failing because `checkout@v3` only gets the last commit.